pennylane/operation.py

```diff
@@ -98,16 +90,9 @@
     Non-Gaussian CV operations and observables are currently only supported via
     the finite-difference method of gradient computation.
```

Comment:
This information doesn't show in the docs

Reply:
No, operations is not a "package" or class a user should ever use, right? So according to the principle of only putting the user interface online and leaving the rest explained in docstrings, this would not go into the docs...

Reply:
Yep, you're right - I meant more that this information should be moved somewhere where it does show up in the docs.